doc/solverinterface.rst

```diff
@@ -107,6 +107,10 @@ All abstract model types inherit from the abstract type ``AbstractMathProgModel`
     with licensing restrictions on the number of concurrent runs.
     Users must discard the model object after this method is invoked.
 
+.. function:: copy(m::AbstractMathProgModel)
```

Comment:
I added something at https://github.com/joaquimg/MathProgBase.jl/blob/master/test/linproginterface.jl what do you think?